scripts/standalone.mk

```diff
@@ -211,11 +211,17 @@ $(PROGRAM_ELF): \
 # Use elf2hex if we're creating a hex file for RTL simulation
 ifneq ($(filter rtl,$(TARGET_TAGS)),)
 .PHONY: software
+ifeq ($(FREEDOM_SDK_TOOLS_PATH),)
```

Comment:
Rather than having this check, would it make sense to have FREEDOM_SDK_TOOLS_PATH default to scripts/elf2hex/install/bin/$(CROSS_COMPILE)-elf2hex, and FS overside this?

Reply:
I don't think so, as this will keep the dependency to having the python elf2hex compiled, so I would rather have it like this.

Will you approve it to get in to 19.08?

Reply:
My immediate concern is we have two different means of producing hex files, without  the ability to know from which tool was used to produce the hex file. Of course,  if users use freedom-e-sdk cmd line, it will be from the python version and FS will use the C++ version. Unfortunately, when user report an issue, they most often don't tell us that. So to unblock FS having this issue, we need to have this. But we should let CX team to get this info some how, until the next release when we only use one version.

Reply:
Since this is very late in the game, how critical is this? Will FS work if we don't have this? Can it goes in the next 19.08 v1 let say.

Reply:
Yes, I am OK with it going into 19.08.1